e2e/src/utils/utils.js

```diff
@@ -120,34 +120,24 @@ export async function isElementVisible(elementId, index) {
     return false
   }
 }
-/**
- * Wait for an element to be visible for at least set amount of time
- * @param {string} elementId testID of the element to wait for
- * @param {number} timeout timeout in milliseconds
- */
-export async function waitForElementId(elementId, timeout = 10 * 1000) {
+
+export async function waitForElementById({ testID, timeout = 10_000, index = 0, tap = false }) {
```

Comment:
object params are good, especially if params are the same time. Since testID is always required, another format we could consider is
```
waitForElementById(testId, { timeout = 10000, index = 0, tap = false } = {})
```
This way we don't need to repeat `{ testID: ... }` everywhere.

Reply:
Adjusted params in 2a627e8df59fff258d72e7c1753e730aee708393!